### PR TITLE
[bun recipe] stubbin' stubbun

### PR DESCRIPTION
**Pull request by sarah11918**
#### What kind of changes does this PR include?

- New or updated content

#### Description

This both "stubs"/scrubs the existing content as it's no longer entirely correct and helpful with Bun's new Astro support AND it marks `i18nReady:false` for the moment, since we expect @TheOtterlord to contribute more content shortly.

**When this page is updated again, please flip back to `true` so it will be translated!**

**Also, any existing Bun translations should be moved to `/old-translations/` so that everyone defaults to the English until the new docs are translated.**

### CODE REVIEW

File: src/content/docs/en/recipes/bun.mdx [hunk 1]
```diff
@@ -2,86 +2,11 @@
...
-```
-
-Bun's build command will output your site to the `dist/` directory.
-
-Then, you can serve your site using the `preview` command:
-
-```bash
-bun preview
-```
+Using Bun with Astro now officially supported but is still very new. Some integrations may not work as expected. Consult [Bun's official documentation for working with Astro](https://bun.sh/guides/ecosystem/astro) for details. Our community is busy at work preparing our own guide for using Bun in Astro!
```
Comment (sasoria) [inline src/content/docs/en/recipes/bun.mdx:10]:
```suggestion
Using Bun with Astro is now officially supported but is still very new. Some integrations may not work as expected. Consult [Bun's official documentation for working with Astro](https://bun.sh/guides/ecosystem/astro) for details. Our community is busy at work preparing our own guide for using Bun in Astro! 
```

Comment (sasoria) [reply src/content/docs/en/recipes/bun.mdx:10]: Awesome, I added a small suggestion.

**Review by sasoria — APPROVED** [review]

File: src/content/docs/en/recipes/bun.mdx (hunk 1 above)
Comment (sarah11918) [inline src/content/docs/en/recipes/bun.mdx:10]:
```suggestion
Using Bun with Astro is now officially supported but is still very new. Some integrations may not work as expected. 

Consult [Bun's official documentation for working with Astro](https://bun.sh/guides/ecosystem/astro) for details. 

## Community Resources

[Hello from Bun](https://blog.otterlord.dev/posts/hello-from-bun/) - blog post
```

Comment (sasoria) [reply src/content/docs/en/recipes/bun.mdx]: It looks like this already is resolved! 🙌

Comment (sarah11918) [reply src/content/docs/en/recipes/bun.mdx]: Thank you @sasoria ! Yup, I caught it myself around the same time!  Nicely spotted!  💜

File: src/content/docs/en/recipes/bun.mdx [hunk 2]
```diff
@@ -2,86 +2,17 @@
...
 type: recipe
-i18nReady: true
+i18nReady: false
 ---
 
-[Bun](https://bun.sh/) is a runtime for JavaScript built for speed. See [Bun's documentation](https://bun.sh/docs) for more information.
+[Bun](https://bun.sh/) is a runtime for JavaScript built for speed.
 
-Using Bun with Astro still is experimental. Some integrations may not work as expected. If you have [any problems using Bun](https://github.com/oven-sh/bun/issues/new/choose), please [open an issue on GitHub](https://github.com/withastro/astro/issues/new/choose)!
+Using Bun with Astro is now officially supported but is still very new. Some integrations may not work as expected. Consult [Bun's official documentation for working with Astro](https://bun.sh/guides/ecosystem/astro) for details.
```
Comment (lilnasy) [inline src/content/docs/en/recipes/bun.mdx:10]: I think this is liable to be misconstrued. Officially supported by whom? It's not us.
```suggestion
Bun recently made their first stable release. However, using Bun with Astro is still very new. Some integrations may not work as expected. Consult [Bun's official documentation for working with Astro](https://bun.sh/guides/ecosystem/astro) for details. 
```

File: src/content/docs/en/recipes/bun.mdx [hunk 3]
```diff
@@ -2,86 +2,17 @@
...
-Using Bun with Astro still is experimental. Some integrations may not work as expected. If you have [any problems using Bun](https://github.com/oven-sh/bun/issues/new/choose), please [open an issue on GitHub](https://github.com/withastro/astro/issues/new/choose)!
+Using Bun with Astro is now officially supported but is still very new. Some integrations may not work as expected. Consult [Bun's official documentation for working with Astro](https://bun.sh/guides/ecosystem/astro) for details. 
 
-## Installing Bun
+If you have any problems using Bun, please [open an Issue on GitHub with Bun directly](https://github.com/oven-sh/bun/issues/new/choose).
 
-Use the following command to install Bun:
+## Community Resources
 
+[Hello from Bun](https://blog.otterlord.dev/posts/hello-from-bun/) - blog post
```
Comment (sarah11918) [inline src/content/docs/en/recipes/bun.mdx:16]:
```suggestion
## Community Resources

Have an article about using Bun with Astro? Add your blog post or video to this page!

[Building a Cloudflare Pages site with Bun](https://blog.otterlord.dev/posts/hello-from-bun/) - blog post
```

File: src/content/docs/en/recipes/bun.mdx (hunk 2 above)
Comment (TheOtterlord) [reply src/content/docs/en/recipes/bun.mdx:10]: +1 although maybe `edge cases` or `incompatibilities` is more granular than `rough edges`?

Comment (lilnasy) [reply src/content/docs/en/recipes/bun.mdx:10]: They are still getting bug reports for fatal crashes, so I would prefer being general.

https://github.com/oven-sh/bun/issues?q=is%3Aissue+is%3Aopen+core+dumped

**Review by TheOtterlord — APPROVED** [review]

**Review by lilnasy — APPROVED** [review]

**Pull request merged**